src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -1409,6 +1407,13 @@ protected WebElement getElementToScrollTo(String place, String container) {
      */
     protected void scrollTo(WebElement element) {
         getSeleniumHelper().scrollTo(element);
+        waitAfterScroll();
+    }
+
+    /**
+     * Wait after the scroll if needed
+     */
+    protected void waitAfterScroll() {
```

Comment:
make `waitAfterScroll` a parameter of this method, so subclasses can also access it.